azure-pipelines.yml

```diff
@@ -167,19 +167,19 @@ jobs:
       - task: Maven@3
         displayName: Set library version
         inputs:
-          mavenPomFile: binding-library\java\pom.xml
+          mavenPomFile: java-library\pom.xml
           goals: versions:set
           options: --define=newVersion=$(version)
 
       - task: Maven@3
         displayName: Build library
         inputs:
-          mavenPomFile: binding-library\java\pom.xml
+          mavenPomFile: java-library\pom.xml
           options: --batch-mode --define=gpg.skip --define=org.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn --update-snapshots
 
       - powershell: |
           $prefix = 'azure-functions-java-library-rabbitmq-$(version)'
-          $source = 'binding-library\java'
+          $source = 'java-library'
           $destination = '$(Build.ArtifactStagingDirectory)\$(version)'
```

Comment:
Updated folder name for Java code to match more with https://github.com/Azure/azure-functions-java-library/ where Java annotations for most of other extensions reside. Also made minor change to build job names. The job names show up on every PR page in PR validation section.